content/influxdb/v2.2/api-guide/iot-starter/python.md

```diff
@@ -0,0 +1,743 @@
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNav">
+            <ul class="navbar-nav">
+                <li class="nav-item">
+                    <a class="nav-link" href="#">About</a>
+                </li>
+                <li>
+                    <a class="nav-link" href="{{url_for('get_buckets')}}">Buckets</a>
+                </li>
+        </div>
+    </nav>
+    <div class="container">
+        {% block content %} {% endblock %}
+    </div>
+
+    <!-- Optional JavaScript -->
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+  </body>
+</html>
+```
+{{% caption %}}[@influxdata/iot-python-app ./templates/base.html](https://github.com/influxdata/iot-python-api/blob/main/templates/base.html){{% /caption %}}
+
+`base.html` provides the layout, navbar, CSS, and JavaScript for your UI.
+To inherit the base layout, all other pages in your application will extend `base.html` and provide
+content to be rendered in the content block:
+
+```html
+  <div class="container">
+        {% block content %} {% endblock %}
+  </div>
+```
+
+
+## Configure IoT App
+
+### Create config.ini
+
+The Python client library allows you to integrate InfluxDB into your application
+and interact with your InfluxDB instance. The client needs the following 
+information to connect to your InfluxDB instance:
+
+
+* your InfluxDB [API token](#authorization) with permission to query (_read_) buckets
+and create (_write_) authorizations for IoT devices.
+* your InfluxDB instance url
+* your InfluxDB org ID
+* your InfluxDB bucket names
+
+To set up the client configuration, create a `config.ini` in your project's top 
+level directory.
+In `config.ini`, paste the following:
+
+```ini
+[APP]
+INFLUX_URL = {{INFLUX_URL}}
+INFLUX_TOKEN = {{INFLUX_TOKEN}}
+INFLUX_ORG = {{INFLUX_ORG_ID}}
+INFLUX_BUCKET = {{INFLUX_BUCKET_FOR_TELEMETRY}}
+INFLUX_BUCKET_AUTH = {{INFLUX_BUCKET_FOR_DEVICES}}
+```
+{{% caption %}}[@influxdata/iot-python-app config.ini](https://github.com/influxdata/iot-python-api/blob/main/config.ini){{% /caption %}}
+
+The following is a sample configuration.
+```ini
+[APP]
+INFLUX_URL = https://us-west-2-2.aws.cloud2.influxdata.com/
+INFLUX_TOKEN = 52Pc_ZkJsRh1PKzlwrK8yO6jWSDh6WPAHbfqp-5aROz4zBnY2mvkKws9YoYzksGH3_Xp90rVqo2PRiajTxaUcw==
+INFLUX_ORG = bea7ea952287f70d
+INFLUX_BUCKET = sly's Bucket
+INFLUX_BUCKET_AUTH = devices_auth
+```
+
+
+## Create IoT Virtual Device
+The IoT virtual device generates simulated weather data for you to store in
+InfluxDB. 
+
+In the `./api` directory of your project, create a `sensor.py` file.
+
+[//]: # Probably preferable to link the file rather than have the whole file written up()
```

Comment:
I'd keep the code here, but to shorten the page, you can wrap it in a `{{% truncate %}}` shortcode. This visually truncates the code block, but lets users expand and collapse it.